Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -956,10 +955,17 @@ def seek_fetchable_offenses(client: QRadarClient, start_offense_id, user_query):
                     )
                 lim_id = lim_offense[0]["id"]  # if there's no id, raise exception
             if lim_id >= end_offense_id:  # increment the search until we reach limit
-                start_offense_id += client.offenses_per_fetch
+                start_offense_id += (client.offenses_per_fetch * tries)
+                tries += 1
+                if tries % 10 == 0:
+                    last_run = get_integration_context(SYNC_CONTEXT)
+                    last_run["id"] = end_offense_id
```

Comment:
I think you might miss incidents this way.
if offense corresponding to end_offense_id was not fetched (was sliced in line 968), the next time this function will be called will miss it

Reply:
It's ok, because then this code will be unreachable (it only runs when there's no fetch), so it's impossible to make it miss offenses, as none were fetched in this timeframe.